### PR TITLE
Update sl_SI translation

### PR DESCRIPTION
**Pull request by petk**
Hello, this patch updates sl_SI translation according to the English origin.

Thank you for considering merging it in the repository.

### CODE REVIEW

**Comment by petk** [comment]
Bumping in hope of getting any kind of response here. Is this repository active? Maybe something needs to be fixed in the PR? Thank you so much for your response and possible merge.

**Comment by petk** [comment]
I'm guessing this is not much of a relevance. Closing...

**Comment by Ocramius** [comment]
@petk this is relevant, but we can't merge it without somebody versed in the language reviewing it

File: languages/sl/Zend_Validate.php [hunk 1]
```diff
@@ -283,6 +297,11 @@
     "The input is less than %min% characters long" => "Vnos je manjši od števila znakov %min%",
     "The input is more than %max% characters long" => "Vnos je daljši od števila znakov %max%",
 
+    // Zend\Validator\Timezone
+    "Invalid timezone given." => "Podano neveljavno časovno območje.",
```
Comment (archseer) [inline languages/sl/Zend_Validate.php]: območje -> pas

https://sl.wikipedia.org/wiki/%C4%8Casovni_pas

**Comment by archseer** [comment]
@petk @Ocramius Apart from that comment I left, LGTM.

**Comment by Ocramius** [comment]
@petk could you reopen? @archSeer thanks for the review!

**Comment by petk** [comment]
@Ocramius done and reopened. I thought this repository is not active anymore. Well great then...
@archSeer fixed. Thanks for helping out.

Thanks for considering merging it and have a nice day.

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
@petk merged, thanks!

**Pull request merged**